9_ninth_hw/2_second_task.py

```diff
@@ -0,0 +1,148 @@
+"""
+Create a class with a description of the worker. Any worker. employees.
+
+I will evaluate the completeness of the described class and will withdraw points for everything unnecessary.
+I want to see clean code. I expect to see clean code with annotations.
+So far, no first-class connections with the second. In all methods, I expect to see docstrings with a sane description.
+"""
+from datetime import datetime
+
+now = datetime.today()
+
+
+class Employee:
+    def __init__(self, __name: str, __second_name: str, __gender: str, __salary: int, __hire_date: str,
+                 __birth_date: str):
+        self.__name = __name
+        self.__second_name = __second_name
+        self.__gender = __gender
+        self.__salary = __salary
+        self.__hire_date = __hire_date
+        self.__birth_date = __birth_date
+
+    @property
+    def name(self):
+        """
+        :return: employee name
+        """
+        return self.__name
+
+    @name.setter
+    def name(self, new_name: str):
+        """
+        :param new_name: new name value
+        :return: new name
+        """
+        self.__name = new_name
+
+    @property
+    def second_name(self):
+        """
+        :return: employee second name
+        """
+        return self.__second_name
+
+    @second_name.setter
+    def second_name(self, new_second_name: str):
+        """
+        :param new_second_name: change value
+        :return: edited employee second name
+        """
+        self.__second_name = new_second_name
+
+    @property
+    def gender(self):
+        """
+        :return: employee gender
+        """
+        return self.__gender
+
+    @gender.setter
+    def gender(self, new_gender: str):
+        """
+        :param new_gender: new gender value
+        :return: new gender
+        """
+        self.__gender = new_gender
+
+    @property
+    def hire_date(self):
+        """
+        :return: employee hire date
+        """
+        return self.__hire_date
+
+    @hire_date.setter
+    def hire_date(self, new_hire_date: str):
+        """
+        :param new_hire_date: new hire date value
+        :return: new hire date
+        """
+        self.__hire_date = new_hire_date
+
+    @property
+    def salary(self):
+        """
+        :return: employee salary
+        """
+        return self.__salary
+
+    @salary.setter
+    def salary(self, new_salary: str):
+        """
+        :param new_salary: new salary value
+        :return: new salary
+        """
+        self.__salary = new_salary
```

Comment:
Looks like I can set negative salary to employee)) it is not good)) he is not a slave))